eos-tech-support/eos-diagnostics

```diff
@@ -435,33 +435,6 @@ function collectTemperatureInfo() {
     }
 }
 
-function collectChromiumPluginsInfo() {
-    let oldVersion = tryReadFile('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/VERSION.txt');
-    if (oldVersion) {
-        oldVersion += ' (via old updater)\n';
-    }
-
-    let flashOutput = 'Flash Version: ';
-    let flashPath = Gio.file_new_for_path('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libpepflashplayer.so');
-    if (flashPath.query_exists(null)) {
-        flashOutput += tryReadFile('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/Flash_VERSION.txt',
-                                   'not available') + '\n';
-    } else {
-        flashOutput += 'not installed\n';
-    }
-
-    let widevineOutput = 'Widevine Version: ';
-    let widevinePath = Gio.file_new_for_path('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libwidevinecdm.so');
-    if (widevinePath.query_exists(null)) {
-        widevineOutput += tryReadFile('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/Widevine_VERSION.txt',
-                                      'not available') + '\n';
-    } else {
-        widevineOutput += 'not installed\n';
-    }
-
-    return flashOutput + widevineOutput;
-}
```

Comment:
Would we not want to adjust the Widevine check for the new path?

Or is the assumption that we can just use the version of the Widevine Flatpak runtime and/or assume Chrome has downloaded the right thing?

Reply:
That is a good point - I'm discussing with Ryan about this as we may end up not having a system-wide version of widevine in the end (as chromium flatpak already downloads it automatically into the user's folder). If we end up having a system-wide version it would make sense to keep this but not so much if local for the user.

Reply:
With latest flatpak the widevine cdm is handled/downloaded by chromium itself and has a per-user installation so let's drop this.